functions/api/search/types.ts

```diff
@@ -0,0 +1,104 @@
+// Types copied from site/search/searchTypes.ts to avoid cross-package imports
```

Comment:
Matthieu recently moved the search types to the types package, so you could import them from there.